### ISSUE TITLE
Unable to get shared domains/zones

### ISSUE THREAD

**Issue by doowb** [dnsimple/dnsimple-developer#547]
I have an account that is a "collaborator" on a domain (the domain management has been shared with my account).
I created an API token, but I'm not able to find the domain/zone through the API.

I get my account id through the "whoami" endpoint and use it in the path, but I keep getting "Domain \`example.dev\` not found" from the API.

The main problem is that I'm trying to use the `go-acme/lego` dnsimple provider (via traefik) and I'm not able to supply a different account id. Otherwise, I could get the domain owner's account id and pass that in.

Is there any other way to use my api token to manage zone records for a "shared" domain?

**Comment by DXTimer** [comment]
@doowb Currently, managing shared domains is not supported through the DNSimple API. Have you considered creating a new [certificate resolver](https://doc.traefik.io/traefik/https/acme/#certificate-resolvers) with the alternative account ID and API key to manage LE certificates for the particular domain?

For future support requests, we recommend reaching out directly to our support team. This will allow us to provide you with more specific and account-related assistance.

**Comment by doowb** [comment]
Thank you @DXTimer for the quick response.

> Currently, managing shared domains is not supported through the DNSimple API.

This is what I needed to know and is why I opened the issue here since I didn't find that information in the documentation. I saw the limitations of shared domains, but there isn't any mention of not being able to use the API.

> Have you considered creating a new [certificate resolver](https://doc.traefik.io/traefik/https/acme/#certificate-resolvers) with the alternative account ID and API key to manage LE certificates for the particular domain?

Yes, this is how I have traefik configured (traefik uses the `go-acme/lego` package to interact with the `dnsimple` api). I'll just get an API key attached to the correct account ID.
